src/common/listctrlcmn.cpp

```diff
@@ -292,7 +292,7 @@ wxItemAttr* wxListCtrlBase::OnGetItemColumnAttr(long item, long WXUNUSED(column)
 // Images support
 // ----------------------------------------------------------------------------
 
-void wxListCtrlBase::SetNormalImages(const wxVector<wxBitmapBundle>& images)
+void wxListCtrlBase::SetNormalImages(const wxVector<wxBitmapBundle> &images)
```

Comment:
Let's please avoid such whitespace-only changes (I can filter it out myself before merging, this is mostly for the future).